### DISCUSSION TITLE
The refactor repo is dead?

### DISCUSSION THREAD

**Discussion by Amleto** [nvim-treesitter/nvim-treesitter#7148] [General]
The refactor repo seems to be unmonitored now. No comments on PRs and issues for about a year?

Is the situation likely to improve?

**Comment by clason** [comment]
Pretty much; (almost) no issues and PRs for about a year. So, no, I don't see much changing in the near future, unless interest steps up. (In particular, there are no plans currently to adapt this to the 1.0 rewrite before release.)
